start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloVFSClassLoader.java

```diff
@@ -358,48 +360,24 @@ public static void printClassPath(Printer out, boolean debug) {
           continue;
         }
 
-        String classLoaderDescription;
-        switch (level) {
-          case 1:
-            classLoaderDescription =
-                level + ": Java System Classloader (loads Java system resources)";
-            break;
-          case 2:
-            classLoaderDescription =
-                level + ": Java Classloader (loads everything defined by java classpath)";
-            break;
-          case 3:
-            classLoaderDescription =
-                level + ": Accumulo Classloader (loads everything defined by general.classpaths)";
-            break;
-          case 4:
-            classLoaderDescription = level + ": Accumulo Dynamic Classloader "
-                + "(loads everything defined by general.dynamic.classpaths)";
-            break;
-          default:
-            classLoaderDescription = level + ": Mystery Classloader ("
-                + "someone probably added a classloader and didn't update the switch statement in "
-                + AccumuloVFSClassLoader.class.getName() + ")";
-            break;
-        }
-
         boolean sawFirst = false;
+        String classLoaderDescription = "Level: " + level + ", Name: " + classLoader.getName()
+            + ", class: " + classLoader.getClass().getName();
         if (classLoader.getClass().getName().startsWith("jdk.internal")) {
           if (debug) {
-            out.print("Level " + classLoaderDescription + " " + classLoader.getClass().getName()
-                + " configuration not inspectable.\n");
+            out.print(classLoaderDescription + ": configuration not inspectable.\n");
```

Comment:
I think there might be an IT that checks the printed strings... not sure. It's fine if it changes, but the IT might need to be updated.